components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/designview/designgenerator/generators/AttributesSelectionConfigGenerator.java

```diff
@@ -60,7 +60,9 @@ public AttributesSelectionConfig generateAttributesSelectionConfig(Selector sele
                 try {
                     selectedAttributes.add(generateSelectedAttribute(outputAttribute));
                 } catch (DesignGenerationException e) {
-                    log.error("Definition of the attribute selection is not found " + e.getMessage(), e);
+                    AllSelectionConfig allSelectionConfig = new AllSelectionConfig();
```

Comment:
why are we doing this while catching the exception ?

Reply:
Because when we select * attributes from a defined stream into an undefined stream, the undefined stream's definition will be created automatically therefore an exception is thrown.